ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -158,16 +158,126 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("concat") {
-			pending("should start subsequent inner signals upon completion") {
+			context("sequencing") {
+				var outerSink: Signal<SignalProducer<Int, NoError>, NoError>.Observer!
+				var outerProducer: SignalProducer<SignalProducer<Int, NoError>, NoError>!
+
+				var previousSink: Signal<Int, NoError>.Observer!
+				var previousProducer: SignalProducer<Int, NoError>!
+
+				var subsequentSink: Signal<Int, NoError>.Observer!
+				var subsequentProducer: SignalProducer<Int, NoError>!
+				var subsequentStarted = false
+
+				beforeEach {
+					outerProducer = SignalProducer { observer, _ in
+						outerSink = observer
```

Comment:
Aaaaaah… I wondered if there was something analogous to `pipe` for `SignalProducer`, yet I've skipped over `buffer` dozens of times on the name alone :see_no_evil: